docs/index.md

```diff
@@ -0,0 +1,120 @@
+---
+---
+# Welcome to kube-hunter documentation
+
+## Documentation for vulnerabilities
+
+For information about a specific vulnerability reported by kube-hunter, enter it's 'VID' (e.g. KHV004) in the search box to the left, to get to the vulnerability article.
+
+## Getting started
+
+### Where should I run kube-hunter?
```

Comment:
I haven't compared word for word but I guess a lot of this is currently in the README - maybe we should reduce duplication and have the README just have the vary bare bones, and refer to the site for most of the content, wdyt? Can be a separate PR if you prefer.

Reply:
Yes this was my thought as well - we should make this one a user friendly guide and the one on github more developer friendly. for now I just wanted to have some content on this page so I copied the readme as is